libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1077,6 +1082,89 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info2[] = {
     // @User: Standard
     AP_GROUPINFO("FONT", 4, AP_OSD_Screen, font_index, 0),
 #endif
+
+#if AP_OSD_CRSF_PANELS_ENABLED      // Parameter and item names are prefixed XF to comply with 16 char limit
+    // @Param: XF_PWR_EN
+    // @DisplayName: XF_PWR_EN
+    // @Description: Displays the TX power when using the CRSF RC protocol
+    // @Values: 0:Disabled,1:Enabled
+
+    // @Param: XF_PWR_X
+    // @DisplayName: XF_PWR_X
+    // @Description: Horizontal position on screen
+    // @Range: 0 59
+
+    // @Param: XF_PWR_Y
+    // @DisplayName: XF_PWR_Y
+    // @Description: Vertical position on screen
+    // @Range: 0 21
+    AP_SUBGROUPINFO(crsf_tx_power, "XF_PWR", 5, AP_OSD_Screen, AP_OSD_Setting),
+
+    // @Param: XF_RSSI_EN
+    // @DisplayName: XF_RSSI_EN
+    // @Description: Displays RC signal strength in dBm for CRSF
+    // @Values: 0:Disabled,1:Enabled
+
+    // @Param: XF_RSSI_X
+    // @DisplayName: XF_RSSI_X
+    // @Description: Horizontal position on screen
+    // @Range: 0 59
+
+    // @Param: XF_RSSI_Y
+    // @DisplayName: XF_RSSI_Y
+    // @Description: Vertical position on screen
+    // @Range: 0 21
+    AP_SUBGROUPINFO(crsf_rssi_dbm, "XF_RSSI", 6, AP_OSD_Screen, AP_OSD_Setting),
+
+    // @Param: XF_SNR_EN
+    // @DisplayName: XF_SNR_EN
+    // @Description: Displays RC signal to noise ratio in dB for CRSF
+    // @Values: 0:Disabled,1:Enabled
+
+    // @Param: XF_SNR_X
+    // @DisplayName: XF_SNR_X
+    // @Description: Horizontal position on screen
+    // @Range: 0 59
+
+    // @Param: XF_SNR_Y
+    // @DisplayName: XF_SNR_Y
+    // @Description: Vertical position on screen
+    // @Range: 0 21
+    AP_SUBGROUPINFO(crsf_snr, "XF_SNR", 7, AP_OSD_Screen, AP_OSD_Setting),
+
+    // @Param: XF_ANT_EN
+    // @DisplayName: XF_ANT_EN
+    // @Description: Displays the current active antenna for CRSF
```

Comment:
```suggestion
    // @Description: Displays the current active antenna
```

similarly elsewhere.